Orange/widgets/visualize/ownomogram.py

```diff
@@ -563,6 +563,7 @@ def add_items(self, items):
 
 class OWNomogram(OWWidget):
     name = "Nomogram"
+    short_name = "Nomogram"
```

Comment:
Isn't `short_name` name optional?

Reply:
`short_name` overrides the text elide and line breaking. The elide is (in my opinion) very aggressive. 
I have not yet checked whether the word fits nicely on Windows and Linux.